scripts/upgrade

```diff
@@ -67,8 +67,8 @@ then
   ynh_secure_remove --file="$tmpdir"
 fi
 
+chmod -R o-rwx "$install_dir"
```

Comment:
this permission is not enough that's why i put back 

`chmod -R 750 "$install_dir"`

Reply:
ok, but then why do you add this `chmod -R o-rwx "$install_dir"` ?

Reply:
I notice on other packages with permission issues that adding `chmod -R o-rwx "$install_dir"` solves the issue...

Reply:
ok, well for æneria, it seems to break the app :/

Nginx can no longer reads public static assets with this permission